invisible_cities/reco/olivia.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import glob
+import os
+
+from collections import defaultdict
+
+import numpy as np
+import json
+
+from   invisible_cities.io .pmaps_io import load_pmaps
+from   invisible_cities.io .hist_io  import save_histomanager_to_file
+from   invisible_cities.evm.histos   import HistoManager
+
+import invisible_cities.reco.histogram_functions as histf
+import invisible_cities.reco.monitor_functions   as monf
+import invisible_cities.database.load_db         as dbf
+
+
+ICDIR = os.environ['ICDIR']
+
+data_types = ['rwf', 'pmaps'  ]
+sources    = ['kr' , 'generic']
+config_files = {'pmaps_kr'      : os.path.join(ICDIR, 'config/PmapsKrConfig.json')     ,
```

Comment:
The city shouldn't know anything about the configuration files available. It should be itself an argument. Actually, the config file should be the only parameter and follow somehow the structure we already have.